src/qt/bitcoingui.cpp

```diff
@@ -1002,7 +1088,10 @@ void BitcoinGUI::gotoVerifyMessageTab(QString addr)
 
 void BitcoinGUI::updateNetworkState()
 {
+    static int nPreviousCount{0};
+    static bool fPreviousState{false};
     int count = clientModel->getNumConnections();
+    bool fNetworkState = clientModel->getNetworkActive();
```

Comment:
nit: imo "state" doesn't fit `bool` very well. Maybe s/`fNetworkState`/`fNetworkActive`/ (and s/`fPreviousState`/`fNetworkActivePrev`/)?

Reply:
Done in af2db17